app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -156,11 +156,19 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
         let offSetDate = new Date();
         offSetDate.setDate(offSetDate.getDate() - offSet);
         offSetDate.setHours(0, 0, 0);
-        const currStartDate = filter.startDate ? new Date(filter.startDate) : offSetDate;
```

Comment:
Previously this would be problematic as filter.dateOffset is undefined when all is selected, the result would be a start and end date of today's date